### ISSUE TITLE
comparison to tf.keras.applications.VGG16

### ISSUE THREAD

**Issue by mistersingh179** [1297rohit/VGG16-In-Keras#3]
Hi, is this the same as the VGG16 application provided by keras 

https://www.tensorflow.org/api_docs/python/tf/keras/applications/VGG16

Can the Keras application be used as-is to train on new data?

**Comment by 1297rohit** [comment]
Yes it is almost same. Official keras one would have some batch normilization layers more.
